docs/source/notes/modules.rst

```diff
@@ -603,10 +603,12 @@ PyTorch provides two types of hooks for modules:
   :func:`~torch.nn.modules.module.register_module_forward_pre_hook` and
   :func:`~torch.nn.modules.module.register_module_forward_hook` functions.
 * **Backward hooks** are called during the backward pass. They can be installed with
-  :func:`~torch.nn.Module.register_full_backward_hook`. These hooks will be called when the backward for this
-  Module has been computed and will allow the user to access the gradients for both the inputs and outputs.
+  :func:`~torch.nn.Module.register_backward_pre_hook` and :func:`~torch.nn.Module.register_full_backward_hook`.
+  These hooks will be called when the backward for this Module has been computed and will allow the user to
+  access the gradients for both the inputs and outputs.
```

Comment:
we should update to say that you'll only see gradient outputs for register_backward_pre_hook